src/to-have-style.js

```diff
@@ -18,7 +18,9 @@ function getStyleDeclaration(document, css) {
 
 function isSubset(styles, computedStyle) {
   return Object.entries(styles).every(
-    ([prop, value]) => computedStyle.getPropertyValue(prop) === value,
+    ([prop, value]) =>
+      computedStyle.getPropertyValue(prop.toLowerCase()) ===
+      value.toLowerCase(),
```

Comment:
I just came across this change with some failing tests after upgrading `@testing-library/jest-dom`. Not a big deal, though I'd like to point out that the value on the right hand side is converted to lowercase, while the one on the left hand side is not. So I guess it should be

```diff
    ([prop, value]) =>
-      computedStyle.getPropertyValue(prop.toLowerCase()) ===
+      computedStyle.getPropertyValue(prop.toLowerCase()).toLowerCase() ===
      value.toLowerCase(),
```

In my case it fails for comparing a `translateX`, so the comparison basically looks like `translateX(0px) vs translatex(0px)` now.

I would have suggest a PR, however, I am not sure if lowercasing the values is a good idea at all, since e.g. `font-family` values and urls are case-sensitive AFAIC. So the question probably is whether we don't lower-case the values at all, or add some exceptions?

Reply:
Maybe it's the `value` what should not be converted to lower case 🤔

Reply:
Hmm this is tricky. I don't think CSS is case-sensitive: `transform: translateX(0px)` works the same as `transform: translatex(30px)`, `TRANSFORM: TRANSLATEX(30px)` and `transform: TRANSLATEx(30px)` (tested in chrome). I guess the best thing to do for now is to use your suggestion and lowercase the `prop`, `value` and `computedStyle.getPropertyValue`.

Reply:
Yeah CSS is case insensitive: https://www.w3.org/TR/2011/REC-CSS2-20110607/syndata.html#characters
>The following rules always hold:
>
>- All CSS syntax is case-insensitive within the ASCII range (i.e., [a-z] and [A-Z] are equivalent), except for parts that are not under the control of CSS. For example, the case-sensitivity of values of the HTML attributes "id" and "class", of font names, and of URIs lies outside the scope of this specification. Note in particular that element names are case-insensitive in HTML, but case-sensitive in XML.
>...

So will need special cases for URIs and font names?

Reply:
So then what we need to do is what @rkusa suggests above. The only question is how to deal with font names and URIs (the only two things that can appear in css property values, because class names and IDs are only used in CSS selectors, which we do not process in `toHaveValues`).

Reply:
Anything preventing us from returning `value.toLowerCase()` to just `value`?

Reply:
Nothing really. I can do it and will do it today. One thing though: I think it's better to add `.toLowerCase()` to the other operand in the comparison, rather than removing it from the `value`. @connormeredith @dylang @rkusa, what do you all think?

Reply:
Since we are talking about tests, the comparison of the values doesn't need to be case insensitive IMHO. So I'd either vote for lower-case with exceptions for fonts and URLs, or for not lowercasing the values at all. Since I can see how adding those exceptions might require quite some work to get it right, I'd be perfectly fine with the latter (not lowercasing the values).

Reply:
Good thinking. Let's avoid the full lowercasing, as it requires the special handling of the exceptions. Thanks for your input.

Reply:
I agree, let's not lowercase the values 👍

Reply:
I very quickly created the PR for this: #163. Just did it fully online and in the browser just to kick it off. It probably need to adjustments in the tests, to which I'll get later today. But also to give some time to discuss there if things work.

BTW, to those of you affected, can you try out this very simple change by directly editing the corresponding file in your `node_modules` and run your tests again? Does it all work as expected when you introduce this change?

Reply:
Done in #164, thanks to @connormeredith. Should go out in a next release (should be 4.2.4, keep an eye for it being announced in that very PR in a while).

Reply:
Awesome, thanks @connormeredith  and @gnapse!